test/unit/warpable_test.rb

```diff
@@ -59,6 +59,6 @@ def setup
     assert File.exist?('public/warps/saugus-landfill-incinerator/folder')
     assert_not_nil @warp.save_dimensions
     assert_not_nil @warp.user_id
-    assert File.exist?('public/warps/saugus-landfill-incinerator/1-geo.tif')
+    # assert File.exist?('public/warps/saugus-landfill-incinerator/1-geo.tif')
```

Comment:
@cesswairimu Why did you have to comment this? I facing the same problem at https://github.com/publiclab/mapknitter/pull/605#issuecomment-493623570

Reply:
@alaxalves @cesswairimu I'm pretty sure this test has always been having issues. Please see my comment on the original PR for it here: https://github.com/publiclab/mapknitter/pull/341#issuecomment-477821230

But I haven't looked into this beyond that since then so this is just a suggestion in case it helps

Reply:
Aha! Thanks for the insight @sashadev-sky, actually I thought I broke something with the upgrade but everything worked fine, that's why I commented it out @alaxalves. Thanks

Reply:
@sashadev-sky Is it ok to open a PR removing this? This test is blocking my work at https://github.com/publiclab/mapknitter/pull/605 too.

Reply:
Hi all! I think this is a remnant from a test of the exporter system, which is here: https://github.com/publiclab/mapknitter/blob/dc96adae1b376e7a90a3410c535dd547230caf5e/test/unit/exporter_test.rb#L37

the geotiff produced should happen after the line I'm linking to, so you could try moving this assertion there. I don't think anything else in this test should be generating such a geotiff, though, so it makes sense that it failed!

Reply:
@jywarren @alaxalves @cesswairimu I spent a bit of time on this and there are two things here:
1) The images don't delete on export - I think that was the expected behavior and we currently have the same tests being run for map tests, warpables tests, and exporter tests, so they are interfering with each other. The reason the error comes up depending on if you run the tests in parallel or not is because of this.

So I am also a bit confused about the end goal. Is it that the map test should just generate the original warpables, and then the exporter tests should do the rest with the file creation, and warpables shouldnt be running any export testing code at all? 

2) I think the reason the test is failing is because of the way we modularized the export code, the functions are attempting to generate the .tif incorrectly. But the reason we still see there are generated .tif files is because those same functions are still being triggered in the map / warpables classes (so basically we didn't remove the old implementation and that is what were seeing work) I got the tests to pass by cleaning up the files a lot so i'm just going to pull into a separate PR so we can all look over it

Reply:
@sashadev-sky That PR would be more than welcome. 
I have been digging those up too, and I got to the same conclusion. The reason of those tests not passing when I split them to run in parallel is because they depend on the previous tests to be ran, since those invoke methods needed for them to pass. This is what I suspect that caused this -> https://github.com/publiclab/mapknitter/issues/599

Reply:
> Hi all! I think this is a remnant from a test of the exporter system, which is here:
> 
> https://github.com/publiclab/mapknitter/blob/dc96adae1b376e7a90a3410c535dd547230caf5e/test/unit/exporter_test.rb#L37
> 
> the geotiff produced should happen after the line I'm linking to, so you could try moving this assertion there. I don't think anything else in this test should be generating such a geotiff, though, so it makes sense that it failed!

I have also tried adding those lines @jywarren provided us:
https://github.com/publiclab/mapknitter/blob/dc96adae1b376e7a90a3410c535dd547230caf5e/test/unit/exporter_test.rb#L37
but this ended confirming my "theory" that the results of previous test influence on the upcoming tests.

Reply:
Just fixed this at https://github.com/publiclab/mapknitter/pull/605/commits/654361e0e451ee3c473efde379c295ab50860b43